### ISSUE TITLE
Error Pushing the OSCAR image to the Docker Registry

### ISSUE THREAD

**Issue by gmolto** [grycap/oscar#5]
Attempting to deploy an OSCAR cluster results in the following error:

```
TASK [grycap.kubeoscar : Build the OSCAR image and push it to the docker registry] ***
fatal: [158.42.104.167_0]: FAILED! => {"changed": false, "msg": "Error pushing image registry.docker-registry/oscar: Get https://registry.docker-registry/v1/_ping: dial tcp 10.106.195.1:443: getsockopt: connection refused"}
```

**Comment by alpegon** [comment]
That happens because ansible it's trying to upload the image to the internal registry before the registry has been deployed.
I'm changing the way OSCAR is deployed in Kubernetes to fix this.

**Comment by alpegon** [comment]
The issue is fixed in the PR #8
